queries.sql

```diff
@@ -32,10 +32,152 @@ WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01'
 GROUP BY species;
 
 SELECT
-    name,
-    full_name,
-    owners_id
+    name as Melody_owns
 FROM
-    owners
+    animals
  JOIN owners
-    ON owners_id = 4;
\ No newline at end of file
+    ON owners.id = owners_id
+    WHERE full_name = 'Melody Pond';
+
+SELECT
+    animals.name as name,
+    species.name as species
+FROM
+    animals
+ JOIN species
+    ON species_id = 1
+    where species.name = 'Pokemon';
+
+    SELECT 
+    owners.full_name,
+    animals.name
+    from owners
+    left JOIN animals
+    ON owners.id = owners_id ;
+
```

Comment:
Hello there 👋🏾 the result of the following queries are incorrect please check your database and try again 👍🏾  thank you

- Who was Maisy Smith's first visit?
- How many times visits were with a vet that did not specialize in that animal's species?

the result of the first query ought to be `boarmon`
and the development of the second ought to be 12 unspecialized visits